storage/src/journal/authenticated.rs

```diff
@@ -1267,9 +1252,10 @@ mod tests {
                 Location::new_unchecked(20),
                 "commit should be at location 20"
             );
-            journal.close().await.unwrap();
+            journal.sync().await.unwrap();
```

Comment:
This is `test_close_with_pending_operations`...do we still want this test now that `close` is no more? I'll leave it for now.